### DISCUSSION TITLE
pulp citation [bibtex]

### DISCUSSION THREAD

**Discussion by ordavidov** [coin-or/pulp#478] [General]
Hi everyone, 

I'd like to cite PuLP in a paper. What's the best citation? If possible, a pointer to the bibtex blurb would be great.

Thanks!

**Comment by stumitchell** [comment]
hi there is this.

http://www.optimization-online.org/DB_FILE/2011/09/3178.pdf

or the docs
https://coin-or.github.io/pulp/

please also cite CBC if that is the solver you are using

https://github.com/coin-or/Cbc they have a citation link on their main page
we should probably do that.

Stu
Stuart Mitchell
PhD Engineering Science
Extraordinary Freelance Programmer and Optimisation Guru
www.stuartmitchell.com

**Reply by ordavidov** [reply]
Thanks a bunch!
